services/notifications/src/modules/kafka/kafka.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Controller, OnModuleInit } from '@nestjs/common';
+import { KafkaService } from './kafka.service';
+import { EventPattern } from '@nestjs/microservices';
+import { TopicEnum } from './enums';
+import { KafkaMessageInterface } from './interfaces';
+
+@Controller()
+export class KafkaController implements OnModuleInit {
+  constructor(private readonly kafkaService: KafkaService) {}
+
+  @EventPattern(TopicEnum.NEW_USER_REGISTERED)
+  handleNewUserRegistered(message: KafkaMessageInterface) {
+    this.kafkaService.handleNewUserRegistered(message?.value);
```

Comment:
Does optional chaning is really needed here `message?.value` ? 
`value` prop seems to be required in `KafkaMessageInterface`